ipaserver/dcerpc_common.py

```diff
@@ -0,0 +1,74 @@
+import six
+from ipalib import _
+if six.PY3:
+    unicode = str
```

Comment:
Please no more ``unicode`` hack in new code. If you need a version dependend text type, please use ``six.text_type``.